src/building-blocks/ButtonToolbar.tsx

```diff
@@ -6,7 +6,6 @@ import { Form } from 'antd';
 
 export interface IButtonToolbarProps {
   align?: 'between' | 'right';
-  children?: any;
```

Comment:
This was useless. No need to define it when extending Component except if you want to force it or force specific children type.